.github/workflows/build.yml

```diff
@@ -10,15 +10,20 @@ jobs:
         ruby: [ '2.6', '2.7', '3.0', '3.1', '3.2' ]
 
     steps:
-    - uses: actions/checkout@v4
-    - name: Set up Ruby ${{ matrix.ruby }}
-      uses: ruby/setup-ruby@v1
-      with:
-        ruby-version: ${{ matrix.ruby }}
-    - name: Build and test with Rake
-      run: |
-        gem install bundler
-        bundle install --jobs 4 --retry 3 --without guard
-        bundle exec rspec spec
-        bundle exec rubocop
+      - uses: actions/checkout@v2
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+      - name: Install Bundler
```

Comment:
Bundler is automatically installed with the `ruby/setup-ruby` action, including with this conditional.